docs/content/manual/advanced/addons/5846-vm-import-enhancement.md

```diff
@@ -0,0 +1,453 @@
+---
+title: VM import with EFI mode and secure boot
+---
+
+* Related issues: [#5846](https://github.com/harvester/harvester/issues/5846) [ENHANCEMENT] vm-import-controller enhancements
+
+## Category: 
+* Virtual Machine
+
+## OpenStack Prerequisite Setup
+1. Prepare a baremetal or virtual machine to host the OpenStack service 
+1. Or use the automated Jenkins pipeline to prepare a devstack cluster (Openstack 16.2) (stable/train)
+1. Install OpenStack command line tool on your local machine (introduce in next section)
+
+## VMware vSphere Prerequisite Setup
+1. Get the available access to the existing or prepared vSphere client (v7.1)
+
+## Harvester Prerequisite Setup
+1. Harvester can connect to the OpenStack dashbord and API endpoint
+1. Harvester can connect to the vShpere client dashboard and API endpoint
+1. Enable the `vm-import-controller` in the `Addons` page
+1. Create the `vlan1` vm network on the `mgmt` interface
+
+## Install OpenStack command line tool
+1. Require python 3.10 or you can use virtual environment (venv)
+  ```
+  virtualenv venv --python=python3.10
+  source venv/bin/activate
+  ```
+1. Install OpenStack client 
+  ```
+  pip install python-openstackclient
+  ```
+1. Download the OpenStack cloud.yaml file 
+   - Access the openstack dashboard
+   - Open `API Access` page
+   - Click the `Download OpenStack RC File` button
+   - Select the `OpenStack cloud.yaml File`
+1. Copy the cloud.yaml file to your openstack config folder 
+  ```
+  mkdir -p ~/.config/openstack
+  cp -v ~/Documents/Harvester/Openstack/clouds.yaml ~/.config/openstack
+  ```
+1. Check can list the image list on OpenStack Server 
+  ```
+  openstack image list --os-auth-url http://172.19.109.182/identity --os-identity-api-version 3 --os-project-name admin --os-project-domain-name default --os-username <username> --os-password <password>
```

Comment:
Avoid IP exposing?

Reply:
Thanks for the check. Remove the IP exposed on the check image command.